.github/workflows/test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           # TODO1 (use uv over pip) uv install torch is flaky, track #3826
           # TODO2 (pin torch version): DGL library (matgl) doesn't support torch > 2.2.1,
           # see: https://discuss.dgl.ai/t/filenotfounderror-cannot-find-dgl-c-graphbolt-library/4302
-          pip install torch==2.2.1
+          pip install --timeout=300 torch==2.2.1
```

Comment:
it seems even `pip install torch` is quite flaky as well, and I guess [it's not to be fixed in the short term](https://github.com/pypi/warehouse/issues/3826#issuecomment-384588312):

> Occasional timeouts will happen.

They recommended bumping the timeout instead (default is 15 s)